spec/fizzbuzz_rspec.rb

```diff
@@ -0,0 +1,31 @@
+require '../solver'
```

Comment:
- [ ] Kindly fix this to require './solver' as its not running tests with '../solver'.
Apply the same changes to factorial and reverse as well.

Reply:
Hi @sja-thedude.
The `solver.rb` file is outside the `spec' folder, i.e., `solver.rb` is in the main directory of our repository. If we did so based on your feedback, It could not work.
We used the command `rspec spec spec/test_file_rspec.rb` in the root directory of our repo., where `test_file_rspec.rb` is the name of the test file.
Thanks